src/karma-webpack.js

```diff
@@ -237,7 +237,11 @@ Plugin.prototype.readFile = function(file, callback) {
   var doRead = function() {
     if (optionsCount > 1) {
       async.times(optionsCount, function(idx, callback) {
-        middleware.fileSystem.readFile(path.join(os.tmpdir(), '_karma_webpack_', String(idx), this.outputs[file]), callback)
+        if (Array.isArray(this.outputs[file])) {
+          middleware.fileSystem.readFile(path.join(os.tmpdir(), '_karma_webpack_', String(idx), this.outputs[file][0]), callback);
```

Comment:
Is it ensured that `this.outputs[file][0]` points to the 'correct'  file?

Reply:
Did you try it it with e.g source maps enabled and/or extracting CSS ?

Reply:
Alright :), I need to rely upon you here since this module has no test 😥

Reply:
Sorry, It seems that it is not always the case for some webpack config :(
So i had to fix this in https://github.com/webpack-contrib/karma-webpack/pull/360